tool/esm_resolver_generator.js

```diff
@@ -38,4 +41,8 @@ function getModuleNames() {
     return paths;
 }
 
+function generateDeclaration() {
```

Comment:
On top level this function names says this is doing something general, but in fact it is generating a very specific declaration, so i'd suggest to move this string inline, where it is used. Also can you use a relative path? and also update pathModules above, which is used for ace-builds .

Reply:
Done